### PR TITLE
World Tables Documentation updates

### PR DESCRIPTION
**Pull request by Kitzunu**
Link fixes

Removed anything that had with versions over WotLK to do

Basically https://github.com/azerothcore/wiki/pull/83 but fixing the conflicting files because I was stupid enough to delete the old fork xD

### CODE REVIEW

**Comment by Kitzunu** [comment]
Note to self: Next to do https://github.com/azerothcore/wiki/pull/83/commits/3b76d40600f848f09e1cd9d27dacb041d8316794

**Comment by Kitzunu** [comment]
I will also correct all links here so Barbz gets happy :P

File: docs/quest_template.md [hunk 1]
```diff
@@ -851,8 +848,7 @@ Time in seconds that the player has to complete this quest.
 
 ### ~~RequiredClasses~~
```
Comment (Kitzunu) [inline docs/quest_template.md]: Does this field exist in AC?

Comment (Kitzunu) [inline docs/quest_template.md]: Does this field exist in AC?

**Comment by Kitzunu** [comment]
@BarbzYHOOL 
@FrancescoBorzi 
wtb reviews <3

File: docs/quest_template.md (hunk 1 above)
Comment (BarbzYHOOL) [reply docs/quest_template.md]: I don't see it in the table but that surprises me

File: docs/pool_pool.md [hunk 2]
```diff
@@ -77,17 +77,18 @@ This is the pool of pools table. You can create a pool with a chance of a range
...
 
-The ID of the pool\_template that defines this "pool of pools".
+The ID of the [pool\_template](pool_template)) that defines this "pool of pools".
 
 ### chance
 
 The explicit percentage chance that this child pool will be spawned.
 
-If the mother pool spawns just one child pool (max\_limit = 1 in the respective mother pool's pool\_template), the core selects the child pool to be spawned in a two-step process: First, only the explicitly-chanced (chance &gt; 0) child pools of the mother pool are rolled. If this roll does not produce any child pool, all the child pools without explicit chance (chance = 0) are rolled with equal chance.
+If the mother pool spawns just one child pool (max\_limit = 1 in the respective mother pool's [pool\_template](http://www.azerothcore.org/wiki/pool_template))), the core selects the child pool to be spawned in a two-step process: First, only the explicitly-chanced (chance &gt; 0) child pools of the mother pool are rolled. If this roll does not produce any child pool, all the child pools without an explicit chance (chance = 0) are rolled with equal chance.
```
Comment (BarbzYHOOL) [inline docs/pool_pool.md]: here is an absolute link, can you change it?

File: docs/quest_poi_points.md [hunk 3]
```diff
@@ -8,28 +8,163 @@ Comes from sniffs. Visually speaking, this table is used to identify the X and Y
...
+<td><p>int(10)</p></td>
+<td><p>unasigned</p></td>
+<td><p>PRI</p></td>
+<td><p>NO</p></td>
+<td><p>0</p></td>
+<td><p></p></td>
+<td><p></p></td>
+</tr>
+<tr class="even">
+<td><p><a href="#quest_poi_points-idx">idx</a>2</p></td>
```
Comment (BarbzYHOOL) [inline docs/quest_poi_points.md]: the 2 is not inside the <a> https://github.com/azerothcore/wiki/blob/137f604de4cb53638b8642bc6da9ff2bcc202a23/docs/quest_poi_points.md

File: docs/outdoorpvp_template.md [hunk 4]
```diff
@@ -1,7 +1,59 @@
...
+<td><p></p></td>
+<td><p></p></td>
+</tr>
+</tbody>
+</table>
+
+**Description of the fields**
+
+### TypeId
+`field-no-description|1`
```
Comment (BarbzYHOOL) [inline docs/outdoorpvp_template.md]: what is that |1 at the end for?

Maybe just add "no description yet - feel free to edit the page"

File: docs/quest_template.md [hunk 5]
```diff
@@ -1213,7 +1205,7 @@ The quest text. You can use certain placeholders that will be filled in in-game:
 
 ### EndText
 
-field-no-description|57
+`field-no-description|57`
```
Comment (BarbzYHOOL) [inline docs/quest_template.md]: we prob don't have this either or it has antoher name

File: docs/quest_template.md [hunk 6]
```diff
@@ -1225,7 +1217,7 @@ Text sent to player when the player tries to talk to the NPC with the quest acti
 
 ### CompletedText
 
-field-no-description|59
+`field-no-description|59`
```
Comment (BarbzYHOOL) [inline docs/quest_template.md]: same here :( i don't find the column

File: docs/quest_template.md [hunk 7]
```diff
@@ -963,7 +955,7 @@ This flag field defines more specifically the type of quest it is. Aside from th
 <tr class="even">
 <td><p>1</p></td>
 <td><p>QUEST_FLAGS_STAY_ALIVE</p></td>
-<td><p>If the player dies, the quest is failed.<img src="images/icons/emoticons/help_16.png" title="(question)" alt="(question)" class="emoticon emoticon-question" /></p></td>
+<td><p>If the player dies, the quest is failed.?</p></td>
```
Comment (BarbzYHOOL) [inline docs/quest_template.md]:
```suggestion
<td><p>If the player dies, the quest is failed.</p></td>
```

Btw I don't think the <p> are needed but well, that's a detail

**Comment by BarbzYHOOL** [comment]
too much to review, you want us dead? :(

**Comment by Kitzunu** [comment]
Hahahha, Ibwill check all in the coming week, I am too busy this weekend

File: docs/quest_template.md (hunk 1 above)
Comment (Kitzunu) [reply docs/quest_template.md]: So remove it from wiki?

File: docs/quest_poi_points.md (hunk 3 above)
Comment (Kitzunu) [reply docs/quest_poi_points.md]: I don't understand

File: docs/outdoorpvp_template.md (hunk 4 above)
Comment (Kitzunu) [reply docs/outdoorpvp_template.md]: I have no idea, it is something that is used on the TC page.

Comment (BarbzYHOOL) [reply docs/outdoorpvp_template.md]: i think it's the number for each column ^^

**Comment by BarbzYHOOL** [comment]
thank you kitzunu

**Pull request merged**